CHANGELOG.md

```diff
@@ -1,314 +1,303 @@
+## [8.0.1](https://github.com/PedroBern/react-native-collapsible-tab-view/compare/v8.0.0...v8.0.1) (2024-07-05)
```

Comment:
@andreialecu sorry I ran prettier on this file and it changed formatting. My main changes were just the 8.0.1 entry